visyn_core/tests/test_telemetry.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from fastapi.testclient import TestClient
+from prometheus_client.parser import text_string_to_metric_families
+
+
+@pytest.mark.parametrize(
+    "workspace_config",
+    [
+        {
+            "tdp_core": {
+                "enabled_plugins": ["tdp_core"],
```

Comment:
```suggestion
            "visyn_core": {
                "enabled_plugins": ["visyn_core"],
```

Is it really still tdp_core?

Reply:
Both works (it falls back to tdp_core if visyn_core key is not found), but more correct is visyn_core. Will change it.